index.html

```diff
@@ -45,12 +45,21 @@ <h1 class="text_header">The best cakes in town <br>deliverd to your door</h1>
         </nav>
 
         </nav>
-        <article class="artical box">
-           <strong> <h1>Welcome</h1>
```

Comment:
You should not use the strong tag for h1 element, use the strong tag for only small sentences or words to highlight them or make certain words stand out from the rest. If you want to make the whole paragraph strong then use css property called "font-weight: bold".

Additionally, the  strong tag is really a child tag and so for that reason, you can't use it as a container for the h1 and p tags, as you did